nowcast/tests/sensorization/test_sensor.py

```diff
@@ -0,0 +1,91 @@
+from datetime import date
+import os
+import tempfile
+from unittest.mock import patch
+
+import numpy as np
+
+from delphi_nowcast.data_containers import LocationSeries, SensorConfig
+from delphi_nowcast.sensorization.sensor import compute_sensors, historical_sensors, _export_to_csv
+
+
+class TestComputeSensors:
+
+    @patch("delphi_nowcast.sensorization.sensor.compute_ar_sensor")
+    @patch("delphi_nowcast.sensorization.sensor.get_indicator_data")
+    def test_compute_sensors_no_covariates(self, get_indicator_data, compute_ar_sensor):
```

Comment:
These arguments should be, e.g., `mock_get_indicator_data`.

Reply:
:+1: fixed